meta-client/src/main/java/org/apache/helix/metaclient/impl/zk/ZkMetaClient.java

```diff
@@ -258,7 +260,12 @@ public boolean subscribeDataChange(String key, DataChangeListener listener,
   public DirectChildSubscribeResult subscribeDirectChildChange(String key,
       DirectChildChangeListener listener, boolean skipWatchingNonExistNode,
       boolean persistListener) {
```

Comment:
Since this API is not published. Can we change it? As we discussed, we will have new API to support persistent listener. Then we can get rid of this boolean option.

Reply:
To make sure I understand, this method is from a public interface from metaclient. Are you proposing to have different APIs for persistent vs non-persistent listener?

Reply:
Right. Since meta client is not published yet. We still have time to add persist & non-persist subscribe APIs

Reply:
May I ask are you suggesting for DirectChildChange, we have 2 separate API explicitly for persist & non-persist subscribe?

Reply:
For the API level change, let's make it a separate PR? Either it's 1 or 2 API, it can be refactored fairly easily after this one is checked in. Thanks.